apps/preview/nuxt/pages/showcases/Search/SearchBasic.vue

```diff
@@ -1,19 +1,20 @@
 <template>
-  <form role="search" @submit.prevent="submit" ref="referenceRef" class="relative">
+  <form ref="referenceRef" role="search" class="relative" @submit.prevent="submit">
     <SfInput
       ref="inputRef"
       v-model="inputModel"
-      @focus="open"
       aria-label="Search"
       placeholder="Search 'MacBook' or 'iPhone'..."
+      @focus="open"
     >
       <template #prefix><SfIconSearch /></template>
       <template #suffix>
         <button
+          v-if="isResetButton"
```

Comment:
```suggestion
          v-if="inputModel"
```
It will be coerced in vue, `v-if` directive will consider any string as true anyway

Reply:
Done [`dd94765`](https://github.com/vuestorefront/storefront-ui/pull/2633/commits/dd947655a0c9321219099c9e688d2808e1cef861)